cranelift/isle/docs/language-reference.md

```diff
@@ -1498,4 +1510,59 @@ newline). The grammar accepted by the parser is as follows:
 <extern> ::= "constructor" <ident> <ident>
            | "extractor" [ "infallible" ] <ident> <ident>
            | "const" <const-ident> <ident> <ty>
+
+<converter> ::= <ty> <ty> <ident>
+
+<spec> ::= "(" <ident> <ident>* <provide> [ <require> ] ")"
```

Comment:
Likewise down here -- let's put all of these in a separate subsection of the grammar, maybe headlined by `# ---- verification extensions ----` or something like that?